Editing (#2)
bkhabs/diem-unified-charter: diem-unified-charter.md

To simplify the beginning work for DIEM, the initial scope is limited
to attested (e.g., an emblem issuer is independent of the asset, but is attesting to
the asset's right to claim protection) and self-signed (e.g., an emblem issuer is the

> entity controlling the protected asset) emblem types and digital assets that must not be able to determine if

Reviewer: ```suggestion
entity controlling the protected asset) emblem types and digital assets that cannot determine if
```
